packages/server/test/integration/file-creation.test.ts

```diff
@@ -0,0 +1,17 @@
+import HttpStatus from 'http-status-codes'
+import request from 'supertest'
+
+const serverPort = process.env.SERVER_PORT || 8000
+const serverURL = `http://localhost:${serverPort}`
+
+describe('File creation', () => {
+  test('should create a file', async () => {
+    const res = await request(serverURL).post('/files').send({
+      created: true
+    })
+    expect(res.status).toEqual(HttpStatus.OK)
+    expect(res.body).toEqual({
+      created: true
+    })
+  })
+})
```

Comment:
Nice first test :)

Reply:
`jest` looks really good for testing, don't have to assemble multiple modules.